### ISSUE TITLE
model not converge

### ISSUE THREAD

**Issue by yilei-wu** [hongweilibran/wmh_ibbmTum#7]
Hi, Very much appreciate your beautiful work!

While runing the train_leave_one_out.py without changing anything, I found the dice loss is always .99x. 
Also, I saw the code sinnipet
{if history.history['loss'][-1] > 0.99:
            model = get_unet(img_shape, first5)
            current_epoch = 1}(line 189 - line 191)
May I ask what's the reason for that ?
Also, would you share your insight about the layer initialization for this taks? (I saw the random initialization is used according to the paper)

**Comment by yilei-wu** [comment]
The author Hongwei(Bran) Li kindly reply my emaill and I will leave it here:

One observation when I trained U-Net with the WMH data is that the model sometimes does not converge due to the randomness of batch training. If the training loss does not decrease after first epoch, then it will not converge forever 😉. That means the model crashes.

Thus I chose to re-create and retrain the model if the loss does not decrease significantly after the first epoch.

And if the model could ‘get through’ the first epoch, it seems to be fine for the rest of the epochs. I normally stopped the training when the loss is around 0.3.
